contributors/design-proposals/cluster-deployment.md

```diff
@@ -125,7 +125,7 @@ network solutions.
 
 1. flannel daemon will run as a standalone binary (not in docker container)
 2. flannel server will assign CIDRs to nodes outside of kubernetes; this will require restarting kubelet
-after reconfiguring network bridge on local machine; this will also require running master nad node differently
+after reconfiguring network bridge on local machine; this will also require running master node differently
```

Comment:
this is trying to distinguish between the master and other nodes, "master node" is not correct